server/verify_test.go

```diff
@@ -36,6 +41,79 @@ import (
 
 var measuredHashes = []crypto.Hash{crypto.SHA1, crypto.SHA256}
 
+func createTpm2EventLog(deviceEnum byte) []byte {
```

Comment:
This function is very similar to the beginning of `TestVerifyAttestationWithSevSnp` , try to reuse the code there as well.

Reply:
Done